$OpenBSD: patch-configure,v 1.4 2017/02/18 10:19:10 jca Exp $

The ffmpeg plugin uses functions from libavcodec, so ensure that it is linked
against libavcodec.

Commit 9bde48f1b59d upstream.

--- configure.orig	Mon Jul 13 12:00:56 2015
+++ configure	Thu Feb  9 12:11:12 2017
@@ -378,7 +378,7 @@ check_aac()
 check_ffmpeg()
 {
 	HAVE_FFMPEG_AVCODEC_H=y
-	pkg_config FFMPEG "libavformat" || return $?
+	pkg_config FFMPEG "libavformat libavcodec" || return $?
 	if check_header "libavcodec/avcodec.h" $FFMPEG_CFLAGS
 	then
 		HAVE_FFMPEG_AVCODEC_H=n
